Chocolate_Vending_Machine.java

```diff
@@ -0,0 +1,240 @@
+package Week_7;
+
+import java.util.*;
+
+public class Chocolate_Vending_Machine implements Cash_Inventory, Item_Inventory {
+	private HashMap<Coin, Integer> cash = new HashMap<>();
+	HashMap<Item, Integer> items = new HashMap<>();

```

Comment:
I'm not too familiar with java, does leaving off the access modifier here make this public?  Would you want anyone to be able to manipulate this?

Reply:
Yes, it is public I need to make it private